### DISCUSSION TITLE
My code is not working please help

### DISCUSSION THREAD

**Discussion by shyamsharma78** [anuraghazra/github-readme-stats#3254] [General]
[![Shyam's GitHub stats](https://github-readme-statds.vercel.app/api?username=shyamsharma78)](https://github.com/shyamsharma78/github-readme-stats)

```md
[![Shyam's GitHub stats](https://github-readme-statds.vercel.app/api?username=shyamsharma78)](https://github.com/shyamsharma78/github-readme-stats)
```

**Comment by DURGESH716** [comment]
Please try this code:

[![Shyam's GitHub stats](https://github-readme-stats.vercel.app/api?username=shyamsharma78)](https://github.com/shyamsharma78/github-readme-stats)

```md
[![Shyam's GitHub stats](https://github-readme-stats.vercel.app/api?username=shyamsharma78)](https://github.com/shyamsharma78/github-readme-stats)
```

**Comment by shyamsharma78** [comment]
Thank you
